### PR TITLE
Mc test

### PR DESCRIPTION
**Pull request by kingmushroom**

### CODE REVIEW

File: TestGenericLibrary.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Test Generic Library <a name='top'>
 [Back](README.md)
 ## Contents
+[Method Assert](#methodassert)<br>
+[Wrangler_Assert](#wranglerassert)<br><br>
+
```
Comment (Piwington) [inline TestGenericLibrary.md]: As we have two sections in Contents, please add the sub headings '### Asserts' and '### Functions' to separate them out.

Comment (kingmushroom) [reply TestGenericLibrary.md]: seems sensible

File: TestGenericLibrary.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Test Generic Library <a name='top'>
 [Back](README.md)
 ## Contents
+### Assertions
+[Method Assert](#methodassert)<br>
+[Wrangler_Assert](#wranglerassert)<br><br>
```
Comment (KrisRogos) [inline TestGenericLibrary.md]: Is there a reason Wrangler_Assert has a '_' when everything else doesn't?

File: es_aws_functions/test_generic_library.py [hunk 3]
```diff
@@ -34,42 +35,51 @@ def wrangler_client_error(lambda_function, runtime_variables,
...
             runtime_variables["RuntimeVariables"]["data"] = test_data
-        with pytest.raises(exception_classes.LambdaFailure) as exc_info:
-            lambda_function.lambda_handler(runtime_variables, context_object)
-        assert "AWS Error" in exc_info.value.error_message
+        assertion(lambda_function, runtime_variables, expected_message)
 
 
-def method_client_error(lambda_function, runtime_variables,
-                        environment_variables, file_name):
+def method_assert(lambda_function, runtime_variables, expected_message):
```
Comment (Piwington) [inline es_aws_functions/test_generic_library.py]: Code Wise All Looks Good Just Requesting That The Method And Wrangler Assert Be Moved So Functions Are In Alphabetic Order Or To Be Above/Below The Other Functions To Separate Them Out.

Comment (kingmushroom) [reply es_aws_functions/test_generic_library.py]: you couldnt leave both comments at the same time?!

Comment (kingmushroom) [reply es_aws_functions/test_generic_library.py]: all done

**Pull request merged**